saritasa_s3_tools/django/views.py

```diff
@@ -0,0 +1,97 @@
+import contextlib
+import dataclasses
+import typing
+
+from django.core.files.storage import default_storage
+from rest_framework import (
+    decorators,
+    permissions,
+    response,
+    status,
+    viewsets,
+)
+from rest_framework.request import Request
+
+import mypy_boto3_s3
+
+from .. import client
+from . import serializers
+
+
+class S3GetParamsView(viewsets.GenericViewSet):
+    """View for getting params for s3 to upload file to S3."""
+
+    serializer_class = serializers.S3RequestParamsSerializer
+    permission_classes = (permissions.AllowAny,)
+
+    @decorators.action(
+        methods=["POST"],
+        url_path="get-params",
+        url_name="get-params",
+        detail=False,
+    )
+    def get_params(
+        self,
+        request: Request,
+    ) -> response.Response:
+        """Get parameters for upload to S3 bucket.
+
+        Current endpoint returns all required for s3 upload data,
+        which should be later sent to `url` as `form-data` url with
+        'file'. Workflow: First, you make request to this endpoint. Then send
+        response data to `url` via `POST` as form-data with file included. In
+        response you will get an url which you can use in API for value of file
+        related fields like avatar for example.
```

Comment:
Maybe here add mentions about ordering is matter for s3 and `file` should be last param